styles/alternativeto/catppuccin.user.css

```diff
@@ -359,6 +359,12 @@
       border-color: @red;
       color: @crust;
     }
+    // warning button
+    [class*="Button_warning"] {
+      background-color: @yellow;
+      border-color: @yellow;
+      color: @crust;
+    }
```

Comment:
Can you also theme this button when `[disabled]`? This is what it looks like right now:

![CleanShot 2024-10-03 at 21 38 35](https://github.com/user-attachments/assets/dc3ec6f3-eb6e-4796-9221-5d6bacd46b04)

Some surface0-ish background color and off-text text color like overlay0/subtext0 would be good I think.

Reply:
As I said before I do not have an account and can't see this one button due to that

Reply:
I just gave you the exact instructions on how to do it 😅

Reply:
```
	&[disabled] {
		background-color: @surface2;
		border-color: @surface2;
		color: @overlay2;
	}
```
That or whatever. Really hard to make a userstyle for a website for which you don't have an account just sayin...

Reply:
oh I'm sorry i didn't see those comments and just modified the html manually
is it fixed now?